travis-ci/managers/fedora.sh

```diff
@@ -48,9 +48,17 @@ for phase in "${PHASES[@]}"; do
             $DOCKER_EXEC meson -Dslow-tests=true build
             $DOCKER_EXEC ninja -v -C build
             $DOCKER_EXEC sh -c "printf '#!/bin/sh\necho The test is failing for unknown reason, skipping; exit 77' >/build/build/test-capability"
-            # Run 'make check'
             $DOCKER_EXEC ninja -C build test
             ;;
+        RUN_ASAN)
+            $DOCKER_EXEC git clean -dxff
+            $DOCKER_EXEC meson -Db_sanitize=address,undefined build
+            $DOCKER_EXEC ninja -v -C build
+            $DOCKER_EXEC sh -c "printf '#!/bin/sh\necho The test is failing for unknown reason, skipping; exit 77' >/build/build/test-capability"
```

Comment:
This one should be removed after https://github.com/systemd/systemd/pull/10687 is merged.

Reply:
It's part of the PR, so we should be fine

Reply:
It's the second copy of the workaround. Kludges tend to sprawl :-) I'll wait for your PR to be merged to remove it altogether and to make sure that the test passes under ASan+UBsan too just in case.

Reply:
Ahh, I see. Thanks!